@xen-orchestra/lite/src/components/host/dashboard/HostDashboardCpuProvisioning.vue

```diff
@@ -3,7 +3,13 @@
     <UiCardTitle>{{ t('cpu-provisioning') }}</UiCardTitle>
     <VtsLoadingHero v-if="!isReady" type="card" />
     <template v-else>
-      <UiProgressBar display-mode="percent" :value="vCpusCount" :max="cpusCount" :legend="t('vcpus')" />
+      <VtsProgressBar
+        :current="vCpusCount"
+        :label="t('vcpus')"
+        :thresholds="cpuProgressThresholds()"
```

Comment:
I think it should be:

```suggestion
        :thresholds="cpuProgressThresholds(t('cpu-provisioning-warning'))"
```